packages/strapi-plugin-upload/admin/src/components/EditForm/index.js

```diff
@@ -172,7 +172,7 @@ const EditForm = forwardRef(
       try {
         const file = await getCroppedResult();
 
-        onSubmitEdit(e, shouldDuplicate, file);
+        onSubmitEdit(e, shouldDuplicate, file, true);
```

Comment:
I'm not sure but why is it always true? It should be the same value as the `shouldDuplicate` parameter?

Reply:
The fourth argument is used to know if the user cropped a file. As we use the same function to update the file info I need more information to emit the correct event.